src/main/java/com/teamwizardry/librarianlib/features/animator/Animation.kt

```diff
@@ -139,5 +150,42 @@ abstract class Animation<T : Any>(val target: T, val property: AnimatablePropert
 
     internal var _id: Int = -1
         private set
+
+    // builder methods
```

Comment:
```suggestion
    //region - Builder methods
```

(with the other suggestion to close the region of course)